.pre-commit-config.yaml

```
@@ -47,6 +47,14 @@ repos:
         stages: [pre-commit]
         require_serial: true
 
+      - id: check-notebooks
+        name: check-notebooks
+        entry: dev/check-notebooks.sh
+        language: system
+        types_or: [jupyter]
```

```suggestion
        files: '\.ipynb$'
```

Let's use `files`. `types_or` is implicit.

---

https://github.com/mlflow/mlflow/pull/14074 to replace `types_or` or `types` in other hooks.